README.md

```diff
@@ -15,15 +15,35 @@
   limitations under the License.
 -->
 
-# Apache Ozone runner base image
+# Apache Ozone **runner** base image
 
-This is the base image to run Apache Hadoop Ozone in docker containers. This is for test/develop and not for production.
+This is the base image to run Apache Hadoop Ozone in docker containers. This is only for test/develop and not for production.
 
-It doesn't include any Ozone specific jar files or release artifacts just any empty environment which includes all the specific tools to run Apache Hadoop Ozone inside containers.
+The container doesn't include any Ozone specific jar files or release artifacts just an empty environment which includes all the specific tools to run and test Apache Ozone inside containers.
 
-To build it, please use:
+The image is available as [apache/ozone-runner](https://hub.docker.com/r/apache/ozone-runner). Build is managed by Docker Hub.
+
+## Development
+
+To build the image, please use:
+
+```
+docker build -t apache/ozone-runner:dev .
+```
+
+To test it with Apache Ozone, do a custom build:
 
 ```
-docker build -t apache/ozone-runner
+mvn clean install -DskipTests -Dskip.npx -DskipShade -o -Ddocker.ozone-runner.version=dev
```

Comment:
I think "offline" mode is not required here, and I'd rather avoid install into local repo.

```suggestion
mvn clean verify -DskipTests -Dskip.npx -DskipShade -Ddocker.ozone-runner.version=dev
```